tinyagentos/routes/chat.py

```
@@ -333,10 +351,12 @@ async def post_message(request: Request):
 
     router_svc = getattr(request.app.state, "agent_chat_router", None)
-    if router_svc is not None:
-        channel = await ch_store.get_channel(channel_id)
-        if channel is not None:
-            router_svc.dispatch(message, channel)
+    if _http_channel is not None:
+        if router_svc is not None:
+            router_svc.dispatch(message, _http_channel)
+        asyncio.create_task(
+            _beads_on_chat_message(request.app, _http_channel, message)
+        )
```

Ruff (0.15.12)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)